roles/config_missing/tasks/main.yml

```
@@ -0,0 +1,77 @@
+---
+- name: configure organizations
+  theforeman.foreman.organization:
+    username: "{{ satellite_admin_username }}"
+    password: "{{ satellite_admin_password }}"
+    server_url: "{{ satellite_server_url }}"
+    name: "{{ item.name }}"
+    state: "{{ satellite_state }}"
+  loop: "{{ satellite_organizations }}"
+  when: satellite_organizations is defined
```

```suggestion
  loop: "{{ satellite_organizations | default([]) }}"
```

I think this allows for less "skipped" tasks which gives nicer output.

---

agree to move this way and away from testing `defined`, I'd actually change the relevant role defaults to `[]` instead of filter, but intentionally made this the same as the existing config role first, not to mix new stuff and refactoring